### ISSUE TITLE
Network does not converge

### ISSUE THREAD

**Issue by FYXX** [twtygqyy/pytorch-SRDenseNet#1]
I use your code, but replaced the dataset, but the loss was volatile and does not converge

**Comment by FYXX** [comment]
The gradient is always 0

**Comment by twtygqyy** [comment]
Hi @FYXX, please check the latest version with PReLu for gradient vanishing. I will push a trained model soon.

**Comment by FYXX** [comment]
Thank you for your help, the network is working. 
PRELU is so amazing~

**Comment by FYXX** [comment]
I trained 60 epochs, psnr = 30.7 (set5, 4x, 100000 images)~~~

**Comment by twtygqyy** [comment]
Did you use Matlab or mat format images for calculating psnr? I will push the model and the test code after new year vacation. Also I'm not sure if my implementation is 100% correct, please let me know if you find any mistake.

**Comment by twtygqyy** [comment]
@FYXX Model uploaded

**Comment by FYXX** [comment]
thank you~
